content/en/docs/Installation/_index.md

```diff
@@ -38,42 +39,96 @@ Scan the new repository for charts.
 helm repo update
 ```
 
+Optionally, show all available chart versions for Kyverno.
+
+```sh
+helm search repo kyverno -l
+```
+
+Based on the manner in which you would like to install Kyverno, see either the [High Availability](#high-availability) or [Standalone](#standalone) options below.
+
+To install the Kyverno [Pod Security Standard policies](/policies/pod-security/) run the below Helm command after Kyverno has been installed.
+
+```sh
+helm install kyverno-policies kyverno/kyverno-policies -n kyverno
+```
+
+### High Availability
+
+The official Helm chart is the recommended method of installing Kyverno in a production-grade, highly-available fashion as it provides all the necessary Kubernetes resources and configurations to meet production needs. By setting `replicaCount=3`, the following will be automatically created and configured as part of the defaults. This is not an exhaustive list and may change. For all of the default values, please see the Helm chart [README](https://github.com/kyverno/kyverno/tree/main/charts/kyverno) keeping in mind the release branch. You should carefully inspect all available chart values and their defaults to determine what overrides, if any, are necessary to meet the particular needs of your production environment.
+
+* Kyverno running with three replicas
+* PodDisruptionBudget
+* Pod anti-affinity configured
+* Ability to exclude Namespaces in the webhook
+
+{{% alert title="Note" color="warning" %}}
+Kyverno does not support two replicas. For a highly-available installation, the only supported count is three.
```

Comment:
Shouldn't any odd number of replicas work? Is there any reason we want to limit to 3 replicas?

Reply:
We have only tested and certified for 3 replicas TMK.

Reply:
I have tested with 4 and 5 replicas. It's work's fine. 
we have problem with 2 replicas, if leader  fails and  till another leader get selected.